src/main/java/mods/eln/Eln.java

```diff
@@ -2606,6 +2605,33 @@ void registerTurbine(int id) {
 			);
 			transparentNodeItem.addDescriptor(subId + (id << 6), desc);
 		}
+
+		{
+			subId = 11;
```

Comment:
Do you think you could start at 12, reserving 11 for the gas turbine?

Just because I already have it in a large world. I'll try to finish it this weekend.